packages/react-dom/src/client/ReactDOMHostConfig.js

```diff
@@ -400,6 +409,30 @@ export const scheduleMicrotask: any =
           .catch(handleErrorInNextTick)
     : scheduleTimeout; // TODO: Determine the best fallback here.
 
+// -------------------
+//     requestAnimationFrame
+// -------------------
+export const supportsFrameEndTask = true;
+export function scheduleFrameEndTask(task) {
```

Comment:
I think "frame aligned" is better than "frame end", it doesn't matters where it fires in the refresh cycle as long as it happens before the next paint

Reply:
Done